app/helpers/claims_helper.rb

```diff
@@ -65,4 +65,8 @@ def academic_years(year_range)
 
     "September 1 #{start_year} - August 31 #{end_year}"
   end
+
+  def formatted_payroll_gender(payroll_gender)
+    payroll_gender == "dont_know" ? "Don’t know" : payroll_gender.humanize
```

Comment:
We can probably use translations for this

Reply:
Why do I never see these things the first time around! Thanks @pezholio! How's this?